### ISSUE TITLE
Add ability to ignore content by doctypeAlias.

### ISSUE THREAD

**Issue by KevinJump** [KevinJump/uSync8#419]
At the moment you can ignore content by path and alias - but the alias is that of the content. 

it would be good if the content could also be ignored by doctype e.g 

```json
"uSync" :{
    "Sets": {
        "Default": {
            "Handlers": {
                "ContentHandler" : {
                    "Settings" : {
                         "IncludeContentTypes" : "customContentTypeAlias",
                         "ExcludeContentTypes": "contentTypeNotToUse"
                    }
                }
            }
        }
    }
}
```

As with other include/exclude behaviours, if you set include you only get the selected values, (so a default is all), and exclude will run after include. 

"RulesOnExport" has to be set if you want them to also run on export.

**Comment by KevinJump** [comment]
Done - will be in next release (e.g. 10.3 / 11.0)

**Comment by KevinJump** [comment]
released in 10.3 https://github.com/KevinJump/uSync/releases/tag/v10.3.0
